core/src/config/tasks.ts

```diff
@@ -22,7 +22,9 @@ export const DEFAULT = {
           ["system", "imports", "runs", "schedules", "groups", "exports"],
           api?.plugins?.plugins
             .filter((plugin) => plugin.apps?.length > 0)
-            .map((plugin) => plugin.apps.map((app) => `exports:${app.name}`)),
+            .flatMap((plugin) =>
+              plugin.apps.map((app) => `exports:${app.name}`)
+            ),
```

Comment:
This was the fix to get the tasks processing properly when a plugin defined more than one app. It used to be returning something like
```
[
   "groups",
   "exports",
   ["exports:hubspot"],
   [ "exports:mailchimp", "exports:mailchimp-oauth"],
   "records"
]
```

and is now returning something like 

```
[
   "groups",
   "exports",
   "exports:hubspot",
   "exports:mailchimp", 
   "exports:mailchimp-oauth",
   "records"
]
```